src/views/AnalysisView.vue

```diff
@@ -87,7 +97,8 @@ const matrix = computed(() => {
     props.scale.head(MAX_SCALE_SIZE).mergeOptions({
       centsFractionDigits: 1,
       decimalFractionDigits: 3,
-    })
+    }), 
+    0
```

Comment:
This whole change should be reverted as I misread your intention when suggesting it.